src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/MigrationJobService.kt

```diff
@@ -79,6 +82,12 @@ class MigrationJobService(
           applicationContext.getBean(TaskDeadlineService::class.java),
           pageSize,
         )
+
+        MigrationJobType.cas2ApplicationsWithAssessments -> Cas2AssessmentMigrationJob(
```

Comment:
How do we make this run?

Reply:
oh interesting... I assumed these ran alongside the other regular migrations but now I see they have to be triggered by a POST request if I understand correctly? 🤔  https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/418v

Reply:
Ah yes. That PR has a doc that tells us how to run them https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/418/files#diff-53a8b6dceb125ee0bda5113152b27fcc0c72bbdc75867bfc3e0ed30bbca17a75R1. It sounds like there could be a post merge task here to run that on dev to double check it. Then to remember to run it on every environment before we're done?

Reply:
hmm this has just made me think... as the dev database gets wiped on every deploy, we'll start [with the seeded data here ](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/595f1e725d8db7c1aac9daac148ec81910cc32c8/src/main/resources/db/seed/local+dev+test/6__cas2_applications.csv)  - so we could test the migration in the dev environment because the seeded data has a few submitted applications without assessments. But then after we've tested we should ensure we create assessments in [the seed job here](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/1a0e54e8d6543a0dd248c434c6d4ca7089bdfa9f/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas2/Cas2ApplicationsSeedJob.kt#L82 ) so that we don't have to keep running the migration every time we deploy.

Reply:
have created a task here so it doesn't get lost if it isn't done before I leave https://dsdmoj.atlassian.net/jira/software/c/projects/CAS2/boards/1347?selectedIssue=CAS2-301